sw/device/tests/sim_dv/sysrst_ctrl_ulp_z3_wakeup_test.c

```diff
@@ -0,0 +1,152 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/dif/dif_sysrst_ctrl.h"
+#include "sw/device/lib/runtime/ibex.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+// Test phase and expected values written by testbench.
+static volatile const uint8_t kTestPhase = 0;
+static volatile const uint8_t kTestExpected = 0;
```

Comment:
does this variable still exist?

Reply:
`kTestExpected` is not used, so I removed it. Thanks for catching that.